packages/compass-shell/src/modules/worker-runtime.ts

```diff
@@ -59,13 +59,13 @@ export function createWorkerRuntime(
       identify: () => {
         /* not needed */
       },
-      // Prefix Segment events with `Shell ` to avoid event name collisions.
+      // Forward Segment events from the shell.
       // We always enable telemetry here, since the track call will
       // already check whether Compass telemetry is enabled or not.
       track: ({ event, properties }) => {
         return track(
-          `Shell ${event}`,
-          properties,
+          'Shell Event',
```

Comment:
Is this still going to be consumable in Segment/Amplitude?

Reply:
asked analytics, this is my speculation, i believe it may require an additional index, after that it would actually be a little more comfortable since they don't need to strip "Shell " from the event (nothing impossible) for cross-tool analysis .. but i don't actually know.

Let's see, in case we can find an alternative, i guess just skipping type checking this one track call here may be ok.

Reply:
Yeah, I'd imagine that `as any` is fine for this one. Plus, if we ever do something similar for the shell, we could go back to having proper typings here pretty easily again

Reply:
Reverted and used a template literal type, thanks @paula-stacho